src/components/policy/src/cache_manager.cc

```diff
@@ -313,6 +313,7 @@ bool CacheManager::AddDevice(const std::string& device_id,
   CACHE_MANAGER_CHECK(false);
   policy_table::DeviceParams& params =
       (*(pt_->policy_table.device_data))[device_id];
+  params.mark_initialized();
 
   // Open SDL stored just device id in policy
   UNUSED(params);
```

Comment:
@AByzhynar Fixed in 88bcff0b28c429c5a1c4f4cc289eac399bbf958b
Now there is no need to do that.